src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -0,0 +1,18 @@
+<div class="ez-versions-list-container">
+    {%  if draftVersions %}
+        <h2 class="ez-list-toolbar-label">{{'locationview.versions.draft.under.edit'|trans({}, 'locationview')}}</h2>
```

Comment:
this `h2` should be inside a `div` with the class `ez-list-toolbar` (valid also for following `h2` as well)